website/addons/figshare/views/config.py

```diff
@@ -38,16 +38,23 @@ def figshare_config_put(node_addon, auth, **kwargs):
     fields = request.json.get('selected', {})
     node = node_addon.owner
 
-    folder = {
-        'name': fields.get('name') or '',
-        'id': fields.get('id') or None,
-        'type': fields.get('type') or None,
-    }
-    node_addon.update_fields(folder, node, auth)
+    name = fields.get('name') or ''
```

Comment:
Related to my previous comment re: `update_fields`: won't `id` and `type` also be set to `None` if the client doesn't provide them?

I'm not sure this is the desired behavior.